src/Juvix/Backends/ArithmeticCircuit/Parameterisation/Integers.hs

```diff
@@ -0,0 +1,75 @@
+module Juvix.Backends.ArithmeticCircuit.Parameterisation.Integers where
```

Comment:
(I think integers are lower-priority for now)

Reply:
Integers are not really integrated to `Parametrisation.hs` but I did code it so I decided to just leave it there for when its time arrives.